pkg/controller/plan/adapter/openstack/common.go

```diff
@@ -0,0 +1,38 @@
+package openstack
+
+import (
+	"fmt"
+	plancontext "github.com/konveyor/forklift-controller/pkg/controller/plan/context"
+)
+
+const (
+	forkliftPropertyOriginalVolumeID = "forklift_original_volume_id"
+)
+
+func getMigrationName(ctx *plancontext.Context) string {
+	return ctx.Migration.GetName()
+}
+func getMigrationID(ctx *plancontext.Context) string {
+	return string(ctx.Migration.GetUID())
+}
+
+func getVmSnapshotName(ctx *plancontext.Context, vmID string) string {
+	const nameFormat = "forklift-migration-vm-%s"
+	return fmt.Sprintf(nameFormat, vmID)
+}
+
+func getSnapshotFromVolumeName(ctx *plancontext.Context, vmID string) string {
+	const nameFormat = "snapshot for %s"
+	return fmt.Sprintf(nameFormat, getVmSnapshotName(ctx, vmID))
+}
+
+func getVolumeFromSnapshotName(ctx *plancontext.Context, vmID, snapshotID string) string {
+	//const nameFormat = "%s-snapshotID-%s"
```

Comment:
what did you mean with this comment? a line below it will look different.

Reply:
Leftovers, will remove it